strings/slurring_drunk_text.json

```diff
@@ -0,0 +1,21 @@
+{
+	"replacements": {
+		"characters": {
+			"common": {
+				"o": "u",
+				"t": "ch",
+				"a": "ah",
+				"u": "oo",
+				"c": " NAR ",
```

Comment:
ah yes, because being drunk makes you say "chiNAR SIE" instead of "tics"

Reply:
you ever get drunk and start espousing nar'sie